packages/cli/src/commands/lockedgold/rewards.ts

```diff
@@ -1,18 +1,18 @@
 import { flags } from '@oclif/command'
-import { BondedDepositAdapter } from '../../adapters/bonded-deposit'
+import { LockedGoldAdapter } from '../../adapters/locked-gold'
 import { BaseCommand } from '../../base'
 import { displaySendTx } from '../../utils/cli'
 import { Flags } from '../../utils/command'
 
 export default class Rewards extends BaseCommand {
-  static description = 'Manage rewards for bonded deposit account'
+  static description = 'Manage rewards for locked Gold account'
```

Comment:
Flagging discrepancy between here and the previous file, "Locked Gold" vs. "locked Gold". Both appear several times throughout.

Reply:
thanks - standardized on Locked Gold

Reply:
Docs still use "locked Gold".